pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/codestyle/xml/VariableNamingConventions.xml

```diff
@@ -146,16 +146,6 @@ public class Foo {
 		]]></code>
     </test-code>
     
-    <test-code>
-        <description>variables in inner classes should not trigger problems in parent declaration</description>
-        <expected-problems>0</expected-problems>
-        <code><![CDATA[
-public class Foo {
-	private static final Bar BAR = new FooBar() { public foo(String arg) { } };
```

Comment:
Hm... could it be, that Apex doesn't support anonymous classes?

Reply:
I couldn't find any doc about them, (and Google doesn't know about "Apex anonymous classes"), so I figured they don't exist. 

I found, that the Jorje class  "Compilation" (which is the supertype for UserClass/UserInterface/etc) has a subtype named "AnonymousClass", but from what I've seen of the parser it looks like that's used for compilation units that are "implicit blocks", so just a sequence of statements (maybe used as scripts idk)